hardhat.config.ts

```diff
@@ -45,14 +45,20 @@ const config: HardhatUserConfig = {
       optimisticEthereum: process.env.OPTIMISM_ETHERSCAN_API_KEY || "",
       optimisticKovan: process.env.OPTIMISM_ETHERSCAN_API_KEY || "",
 
-      // arbitrum
-      arbitrumOne: process.env.ARBISCAN_API_KEY || "",
-      arbitrumGoerli: process.env.ARBISCAN_API_KEY || "",
-
       // polygon
       polygon: process.env.POLYSCAN_API_KEY || "",
       polygonMumbai: process.env.POLYSCAN_API_KEY || "",
     },
+    customChains: [
+      {
+        network: "arbitrumGoerli",
```

Comment:
Reading through the docs here: https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-etherscan#adding-support-for-other-networks it makes it sound like we might need an entry for `etherscan.apiKey.arbitrumGoerli`?

Reply:
oh yeah, good catch. so far as i can tell, there's no way to programmatically verify the contract on arbitrum goerli yet... i think they'll have to add it to the main etherscan explorer vs. just blockscout. i added the entry though

Reply:
looks like a need a re-approval